sdk_extensions/zpages/src/main/java/io/opentelemetry/sdk/extensions/zpages/TraceConfigzZPageHandler.java

```diff
@@ -0,0 +1,343 @@
+
+import io.opentelemetry.sdk.trace.Samplers;
+import io.opentelemetry.sdk.trace.TracerSdkProvider;
+import io.opentelemetry.sdk.trace.config.TraceConfig;
+import java.io.OutputStream;
+import java.io.PrintStream;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+final class TraceConfigzZPageHandler extends ZPageHandler {
+  private static final String TRACE_CONFIGZ_URL = "/traceconfigz";
+  private static final String TRACE_CONFIGZ_NAME = "TraceConfigZ";
+  private static final String TRACE_CONFIGZ_DESCRIPTION =
+      "TraceConfigZ displays information about the current active tracing configuration"
+          + " and allows users to change it";
+  private static final String QUERY_STRING_ACTION = "action";
+  private static final String QUERY_STRING_ACTION_CHANGE = "change";
+  private static final String QUERY_STRING_ACTION_DEFAULT = "default";
+  private static final String QUERY_STRING_SAMPLING_PROBABILITY = "samplingprobability";
+  private static final String QUERY_STRING_MAX_NUM_OF_ATTRIBUTES = "maxnumofattributes";
+  private static final String QUERY_STRING_MAX_NUM_OF_EVENTS = "maxnumbofevents";
+  private static final String QUERY_STRING_MAX_NUM_OF_LINKS = "maxnumoflinks";
+  private static final String QUERY_STRING_MAX_NUM_OF_ATTRIBUTES_PER_EVENT =
+      "maxnumofattributesperevent";
+  private static final String QUERY_STRING_MAX_NUM_OF_ATTRIBUTES_PER_LINK =
+      "maxnumofattributesperlink";
+  // Background color used for zebra striping rows in table
+  private static final String ZEBRA_STRIPE_COLOR = "#e6e6e6";
+  private static final Logger logger = Logger.getLogger(TraceConfigzZPageHandler.class.getName());
+  private final TracerSdkProvider tracerProvider;
+
+  TraceConfigzZPageHandler(TracerSdkProvider tracerProvider) {
+    this.tracerProvider = tracerProvider;
+  }
+
+  @Override
+  public String getUrlPath() {
+    return TRACE_CONFIGZ_URL;
+  }
+
+  @Override
+  public String getPageName() {
+    return TRACE_CONFIGZ_NAME;
+  }
+
+  @Override
+  public String getPageDescription() {
+    return TRACE_CONFIGZ_DESCRIPTION;
+  }
+
+  /**
+   * Emits CSS styles to the {@link PrintStream} {@code out}. Content emited by this function should
+   * be enclosed by <head></head> tag.
+   *
+   * @param out the {@link PrintStream} {@code out}.
+   */
+  private static void emitHtmlStyle(PrintStream out) {
+    out.print("<style>");
+    out.print(ZPageStyle.style);
+    out.print("</style>");
+  }
+
+  /**
+   * Emits the change tracing parameter table to the {@link PrintStream} {@code out}.
+   *
+   * @param out the {@link PrintStream} {@code out}.
+   */
+  private static void emitChangeTable(PrintStream out) {
+    out.print("<table style=\"border-spacing: 0; border: 1px solid #363636;\">");
+    out.print("<tr class=\"bg-color\">");
+    out.print(
+        "<th colspan=2 style=\"text-align: left;\" class=\"header-text\">"
+            + "<b>Permanently change</b></th>");
```

Comment:
The word "Permanently" here doesn't seem right. It's not permament...you can just use this page to change it again, correct?

Reply:
Yea, I followed the wording of OpenCensus, and "permanently" meant that the changes is applied and will not automatically change back to default after a certain amount of time. I don't know if OpenCensus wanted to have a "temporary" change as well. I can remove `Permanent`

Reply:
How about "Update Active TraceConfig"

Reply:
also, if your service gets restarted, it'll go back to whatever it was before, so it's really not very Permanent. :)

Reply:
"Update active TraceConfig" sounds perfect, thanks!